src/StaticLS/HieView/Name.hs

```diff
@@ -32,7 +33,7 @@ newtype ModuleName = ModuleName {name :: GHC.ModuleName}
   deriving (Show, Eq)
 
 instance Hashable ModuleName where
-  hashWithSalt salt = hashWithSalt @Int salt . GHC.Types.Unique.getKey . GHC.getUnique . (.name)
+  hashWithSalt salt = hashWithSalt @Word64 salt . fromIntegral . GHC.Types.Unique.getKey . GHC.getUnique . (.name)
```

Comment:
Casting to a Word64 instead of Int to avoid loosing information in case static-ls is compiled in a 32 bit machine.